tuf/sig.py

```diff
@@ -377,3 +367,106 @@ def generate_rsa_signature(signed, rsakey_dict):
   signature = tuf.keys.create_signature(rsakey_dict, signed)
 
   return signature
+
+
+
+
+
+def sign_over_metadata(
+    key_dict, data,
+    metadata_format=tuf.conf.METADATA_FORMAT):
+  """
+  Higher level function that wraps tuf.keys.create_signature, and works
```

Comment:
I agree in general. Fixing testing on this fork is a task that I do not think is worth the energy right now, as it would be time-consuming and require re-working in the merge anyway. (I could be wrong - lmk if you disagree.)

As for the labels, the docstring directs one to the lower level create_signature docstring... but you're entirely right; this is inadequate. I'll add proper documentation.